spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkDistributionAndOrderingUtil.java

```diff
@@ -218,7 +218,7 @@ private static Distribution buildPositionDeleteUpdateDistribution(
   }
 
   public static SortOrder[] buildPositionDeltaOrdering(Table table, Command command) {
-    if (command == DELETE || command == UPDATE) {
```

Comment:
This is purely to avoid test failures for now. I will rework distribution and ordering in a follow-up.